app/javascript/components/miq-toolbar.jsx

```diff
@@ -45,6 +45,40 @@ const getParams = (urlParms, sendChecked) => {
   return _.filter(params).join('&') || undefined;
 };
 
+const convertMultParamsToArray = (fullParamString) => {
+  // If the given string has multiple valus for one key, the values will overwrite eachother. 
+  // This method will divide the incoming string into key-valur pairs, 
+  // group all key-value pairs that have the same key (ei on key with multiple values), 
+  // then finaly convert the resulting object into a new string that 
+  // notes which keys have multiple of values (so we dont loose data).
```

Comment:
```suggestion
  // If the given string has multiple values for one key, the values will overwrite each other. 
  // Rails has the ability to handle multi-value params if those keys are suffixed with []
  // This method will divide the incoming string into key-value pairs, group all key-value
  //   pairs that have the same key, then finally convert the resulting object into a new
  //   param string that highlights (using []) which keys have multiple values.
```

Reply:
Also, not sure what the convention for method comments is in javascript, but typically the method comment goes before the method signature.